pupil_src/shared_modules/calibration_routines/fingertip_calibration/fingertip_calibration.py

```diff
@@ -0,0 +1,261 @@
+'''
+(*)~---------------------------------------------------------------------------
+Pupil - eye tracking platform
+Copyright (C) 2012-2018 Pupil Labs
+
+Distributed under the terms of the GNU
+Lesser General Public License (LGPL v3.0).
+See COPYING and COPYING.LESSER for license details.
+---------------------------------------------------------------------------~(*)
+'''
+
+import os
+import cv2
+import numpy as np
+from platform import system
+import audio
+from pyglui import ui
+from pyglui.cygl.utils import draw_points, draw_polyline, draw_progress, RGBA
+from pyglui.pyfontstash import fontstash
+from pyglui.ui import get_opensans_font_path
+from ..calibration_plugin_base import Calibration_Plugin
+from ..finish_calibration import finish_calibration
+import torch
+from .models.unet import UNet
+from .models.ssd_lite import build_ssd_lite
+
```

Comment:
Please use the fully qualified import instead of relative imports as I mentioned above.